storybook/stories/components/table.stories.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, object, boolean } from '@storybook/addon-knobs';
+import '@availity/mock';
+import 'availity-uikit';
```

Comment:
The uikit is being imported in `config.scss` for storybook so we can remove the import here